ocaml-lsp-server/test/e2e/__tests__/textDocument-diagnostics.ts

```diff
@@ -53,7 +53,6 @@ describe("textDocument/diagnostics", () => {
               },
             ],
             "uri": "file:///test.ml",
-            "version": 0,
```

Comment:
Why removing the `version`? Aren't you reverting part of #445?

Reply:
Yes, I am. Unfortunately, I don't see how it can work with diagnostics sourced from dune. The issue is that lsp requires you to send all diagnostics for a document in a single notification, but only the notifications from merlin depend on the source. The notifications from dune are "as is" and are computed based on the source on the file system and hence applying a version to them doesn't make sense. Lsp doesn't permit versioning only a subset of diagnostics, so the entire thing has to go.